### PR TITLE
#161254610 setup redux folder/file structure

### PR DESCRIPTION
**Pull request by abadojack**
#### What does this PR do?
- setup redux folder/file structure redux

#### Description of Task to be completed?
- Create the appropriate folder/file structure for React/Redux workflow

#### How should this be manually tested?
- Clone this repository 
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
``` cd ah-orcas-frontend```
- Checkout the setup redux branch
``` git checkout ch-setup-redux-161254610```
- Install the required dependencies:
```npm install```
- Run the local development server:
```npm start```
- You should be able to view the ah-orcas-frontend-application on your browser

#### Any background context you want to provide?
- The current folder structure is the one that was bootstrapped by create-react-app. Creating the React/Redux folder structure makes the code more readable and easier to maintain.

#### What are the relevant pivotal tracker stories?
[#161254610](https://www.pivotaltracker.com/story/show/161254610)

#### Checklist:
- [X] My code follows the style guidelines of this project
- [X] At least 2 people have reviewed my PR
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] My PR has one commit.

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { createStore, applyMiddleware } from 'redux';
+import { createLogger } from 'redux-logger';
+import allReducers from '../reducers';
+
+const logger = createLogger();
+
+export const store = createStore(allReducers, applyMiddleware(logger));
```
Comment (Gidraf) [inline src/store/index.js]: Why have you used logger instead of thunk in middleware??

**Review by Gidraf — CHANGES_REQUESTED** [review]

File: src/store/index.js (hunk 1 above)
Comment (abadojack) [reply src/store/index.js]: I have fixed this. I'm using `thunk` now

**Review by Gidraf — APPROVED** [review]
LGTM

**Review by kipropbrian — APPROVED** [review]
This works as expected.

**Review by evamaina — APPROVED** [review]
LGTM

**Pull request merged**